doc/steps.md

```diff
@@ -66,6 +66,7 @@ Some of useful variables always establishes during role preparation, so them can
   - `dirs_to_remove_on_cleanup` - dirs that will be removed on instance cleanup;
 - `single_instances_for_each_machine` - list of instances (Ansible hosts), one for each physical machine,
   for example, can be used in `delegate_to`;
+- `instances_from_same_machine` - list of instances that on the same physical machine;
```

Comment:
Seems that it's a map, not a list.

Reply:
Maybe add a little example? Like "Use `instances_from_same_machine[inventory_hostname]` to get a list of instances placed on the same machine as the current one".